src/event.rs

```diff
@@ -0,0 +1,11 @@
+use accesskit::Vec2;
+
+pub enum Event {
+    ActionRequest(accesskit::ActionRequest),
+    Simulated(SimulatedEvent),
+}
+
+pub enum SimulatedEvent {
```

Comment:
docstrings!

```suggestion
pub enum Event {
    ActionRequest(accesskit::ActionRequest),
    Simulated(SimulatedEvent),
}

pub enum SimulatedEvent {
```

Reply:
What suggestion did you want to make? There are no changes

Reply:
That was a mistake - but docstrings :)